bubble-wrap.yaml

```diff
@@ -852,6 +878,37 @@ layers:
                 transition:
                     [show, hide]:
                         time: 0s
+                        
+    # Used by the SDK to place point, line, and polygon overlays on the map
+    mz_default_point:
+        data: { source: mz_default_point }
+        draw:
+            sdk-point-overlay:
+                interactive: true
+                sprite: search-active
+                size: [36px,54px]
+                collide: false
+                anchor: top
+                transition:
+                    [show, hide]:
+                        time: 0s
+    mz_default_line:
+        data: { source: mz_default_line }
+        draw:
+            sdk-line-overlay:
+                color: '#06a6d4'
+                order: 503
+                width: 3px
+    mz_default_polygon:
```

Comment:
` order: 501` is a magic number, and the "underlay" version would similarly be a magic number. 

For the SDK: Would the new v0.7 global syntax help us here – so the SDK could modify the order to the "underlay" value, without needing to know it's specific value? 

For people like John: seems like we should have a "underlay" default and "overall" style defaults for them to use that would have the right properties out of the box?